texinfo/org-trello.org

```diff
@@ -603,16 +616,25 @@ remove specific existing org-trello keybindings:
 :DESCRIPTION: Customizing Variables
 :END:
 
-** TODO The Format of org-trello
-This section describes the format of the text in an org-trello buffer.
+** TODO Example Emacs-Lisp Customization
+:PROPERTIES:
+:ALT_TITLE: Example
+:DESCRIPTION: Example Emacs-Lisp Customization
+:END:
+
+** Automatic org-trello Files
+:PROPERTIES:
+:ALT_TITLE: Auto Files
+:DESCRIPTION: Automatic org-trello Files
+:END:
 
-* Contributing to org-trello
+* CHEK Development of org-trello
```

Comment:
CHECK?

I see that you use CHEK consistently everywhere (at least in this commit).
What does that convey?